doc/tutorials/enable_s5.rst

```diff
@@ -160,21 +160,13 @@ The procedure for enabling S5 is specific to the particular OS:
 How to test
 ***********
 
-.. note:: The :ref:`CBC <IOC_virtualization_hld>` tools and service installed by
-   the `software-defined-cockpit
-   <https://github.com/clearlinux/clr-bundles/blob/master/bundles/software-defined-cockpit>`_ bundle
-   will conflict with the vUART and hence need to be masked.
+.. note:: Depends on the rootfs you used for SOS and UOS; system service maybe conflict with the vUART and hence need to be masked.
+   Following is example for Yocto and Ubuntu rootfs(append i915.modeset=0 video=efifb:off in sos kernel cmdline for gvt_d):
```

Comment:
This comes a bit out of the blue. What service _may_ be conflicting with `ttyS1`?

Why are we specifying the Service VM kernel command-line options here? Are these different that we typically use and people are expected at this point to update their system? Can you also explain what these do?

Is a serial port *required* here? I read somewhere that these options should actually be removed if the system does not have an active serial port?